### DISCUSSION TITLE
Flow x13 disable trackpad in tablet mode

### DISCUSSION THREAD

**Discussion by narker** [seerge/g-helper#381] [Ideas]
Request a feature that disables the trackpad when the laptop is flipped into tablet mode, and re-enables it when flipped back to laptop mode.

**Comment by seerge** [comment]
@narker try this build and post log here please

[GHelper.zip](https://github.com/seerge/g-helper/files/11481960/GHelper.zip)

Further discussion under this issue (created by other user) https://github.com/seerge/g-helper/issues/383
